src/main/java/com/vinylteam/vinyl/entity/Offer.java

```diff
@@ -107,12 +107,12 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (!(o instanceof Offer)) return false;
         Offer offer = (Offer) o;
-        return id == offer.id && uniqueVinylId == offer.uniqueVinylId && shopId == offer.shopId && Double.compare(offer.price, price) == 0 && inStock == offer.inStock && Objects.equals(currency, offer.currency) && Objects.equals(genre, offer.genre) && Objects.equals(catNumber, offer.catNumber) && Objects.equals(offerLink, offer.offerLink);
+        return id == offer.id && uniqueVinylId == offer.uniqueVinylId && shopId == offer.shopId && Double.compare(offer.price, price) == 0 && inStock == offer.inStock && Objects.equals(currency, offer.currency) && Objects.equals(catNumber, offer.catNumber) && Objects.equals(offerLink, offer.offerLink);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(id, uniqueVinylId, shopId, price, currency, genre, catNumber, inStock, offerLink);
+        return Objects.hash(id, uniqueVinylId, shopId, price, currency, catNumber, inStock, offerLink);
     }
 
```

Comment:
why to not use Lombok? Won't code more readable?

Reply:
Let`s try, we are always able to rollback changes in case it will not appear as readable as expected